packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardRenderer.tsx

```diff
@@ -16,7 +16,7 @@ export const CloudPulseDashboardRenderer = React.memo(
     const { dashboard, filterValue, timeDuration } = props;
 
     const selectDashboardAndFilterMessage =
-      'Select Dashboard and filters to visualize metrics.';
+      'Select dashboard and filters to visualize metrics.';
```

Comment:
Can we consult the tech writer about this suggestion?
```suggestion
      'Select a dashboard and filters to visualize metrics.';
```